stories/presetCalendar.story.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { object } from "@storybook/addon-knobs";
+import PresetCalendar from "../src/components/PresetCalendar";
+import { action } from "@storybook/addon-actions";
+import { colors } from "../src/theme";
+import { css } from "emotion";
+
+storiesOf("Recipes/PresetCalendar", module).add("Default", () => (
+  <PresetCalendar
```

Comment:
PresetCalendar should extend on Calendar props rather than taking it as an object. calendarProps or anything like that shouldn't be considered for general use. It's just to be used in edge cases.